Description: Remove rpath and trim libraries
 Don't output any rpath-related flags in sane-config --ldflags or via
 pkg-config and trim the libraries to what's needed for libsane (aka dll
 backend).
Author: Julien BLACHE <jblache@debian.org>

Index: sane-backends-1.0.22/tools/sane-backends.pc.in
===================================================================
--- sane-backends-1.0.22.orig/tools/sane-backends.pc.in	2010-12-02 00:49:58.000000000 +0100
+++ sane-backends-1.0.22/tools/sane-backends.pc.in	2011-02-14 19:02:36.205016002 +0100
@@ -2,8 +2,8 @@
 exec_prefix=@exec_prefix@
 libdir=@libdir@
 includedir=@includedir@
-ldflags=@LDFLAGS@ @GPHOTO2_LDFLAGS@
-libs=@LIBS@ @DL_LIBS@ @LIBV4L_LIBS@ @MATH_LIB@ @TIFF_LIBS@ @JPEG_LIBS@ @GPHOTO2_LIBS@ @SOCKET_LIBS@ @AVAHI_LIBS@ @USB_LIBS@ @SCSI_LIBS@ @RESMGR_LIBS@
+ldflags=@LDFLAGS@
+libs=@LIBS@
 
 Name: SANE Backends
 Description: Backends for SANE, the universal scanner interface
Index: sane-backends-1.0.22/tools/sane-config.in
===================================================================
--- sane-backends-1.0.22.orig/tools/sane-config.in	2010-12-02 00:49:58.000000000 +0100
+++ sane-backends-1.0.22/tools/sane-config.in	2011-02-14 19:02:36.205016002 +0100
@@ -7,13 +7,13 @@
 PACKAGE="@PACKAGE@"
 scriptname="sane-config"
 
-LINKER_RPATH="@LINKER_RPATH@"
+LINKER_RPATH=""
 
 prefix="@prefix@"
 exec_prefix="@exec_prefix@"
 
-LDFLAGS="@LDFLAGS@ @GPHOTO2_LDFLAGS@"
-LIBS="@LIBS@ @DL_LIBS@ @LIBV4L_LIBS@ @MATH_LIB@ @TIFF_LIBS@ @JPEG_LIBS@ @GPHOTO2_LIBS@ @SOCKET_LIBS@ @AVAHI_LIBS@ @USB_LIBS@ @SCSI_LIBS@ @RESMGR_LIBS@"
+LDFLAGS="@LDFLAGS@"
+LIBS="@LIBS@"
 pkgincludedir="@pkgincludedir@"
 pkglibdir="@pkglibdir@"
 includedir="@includedir@"
